### PR TITLE
debug: enabled remote debugging with wdb

### PR DESCRIPTION
**Pull request by jbenito3**
- Adds a needed environment variable to wsgi in order to be able to use
  wdb https://github.com/Kozea/wdb

Signed-off-by: Jose Benito Gonzalez Lopez jose.benito.gonzalez@cern.ch

### CODE REVIEW

**Comment by jbenito3** [comment]
@drjova I cannot modify the labels for this one. It is not deployed but we can say it is a no_invenio_pr

File: modules/webstyle/lib/invenio.wsgi [hunk 1]
```diff
@@ -20,6 +20,10 @@
 mod_wsgi Invenio application loader.
 """
 
+# enable remote debuggin with wdb <https://github.com/Kozea/wdb>
+import os
+os.environ['WDB_NO_BROWSER_AUTO_OPEN'] = 'True'
+
```
Comment (ludmilamarian) [inline modules/webstyle/lib/invenio.wsgi]: I would be in favour of enabling the debugger for the development site, not for production (so to move this under the `if CFG_DEVEL_SITE:`)

**Comment by jbenito3** [comment]
@ludmilamarian changes done

File: modules/webstyle/lib/invenio.wsgi [hunk 2]
```diff
@@ -20,6 +20,11 @@
...
 """
 
+# enable remote debuggin with wdb <https://github.com/Kozea/wdb>
+if CFG_DEVEL_SITE:
+    import os
+    os.environ['WDB_NO_BROWSER_AUTO_OPEN'] = 'True'
+
 # start remote debugger if appropriate:
 from invenio.config import CFG_DEVEL_SITE
 if CFG_DEVEL_SITE:
```
Comment (ludmilamarian) [inline modules/webstyle/lib/invenio.wsgi:25]: Can the lines go here? (there is already an `if CFG_DEVEL_SITE`)

Comment (jbenito3) [reply modules/webstyle/lib/invenio.wsgi:25]: ups...too fast...

**Comment by jbenito3** [comment]
@ludmilamarian sorry about the last commit...too fast and I missed the fact. What do you think now? 🛡

**Comment by ludmilamarian** [comment]
Looks good, thanks! Same comment about the copyright years for the next time.

**Pull request merged**